src/lib/mina_numbers/nat.ml

```diff
@@ -29,9 +29,7 @@ struct
     Checked.map (to_bits t) ~f:(fun bits ->
         Random_oracle.Input.Legacy.bitstring bits )
 
-  let constant n =
-    Field.Var.constant
-      (Bigint.to_field (Bigint.of_bignum_bigint (N.to_bigint n)))
+  let constant n = Field.Var.constant (Field.of_int (N.to_int n))
```

Comment:
This is not equivalent. An OCaml `int` is 63 bits, but we use `UInt64` as one of the instantiations.

Reply:
ooh shoot. I'll just revert and revisit later, loading time impact is small